### ISSUE TITLE
RPC calls sometimes return an NCElement instead of an RPCReply

### ISSUE THREAD

**Issue by pakkitzoomzoom** [ncclient/ncclient#593]
https://github.com/ncclient/ncclient/blob/673da3c58e7767a550faf2b1804e17f29d6fdcfc/ncclient/operations/rpc.py#L377

Breaks crossvendor code in situations like this:

```
def get(m, filter=None):
    logger.debug('Getting %s', str(filter))
    if filter and len(filter) > 0:
        c = m.get(filter=('subtree', filter)).data
    else:
        c = m.get().data
    return c
```

Because RPCReply has data attribute but NCElement doesnt. Also, docs say it should be an RPCReply:

In synchronous mode, blocks until the reply is received and returns :class:`RPCReply`. Depending on the :attr:`raise_mode` a `rpc-error` element in the reply may lead to an :exc:`RPCError` exception.

Originally from https://github.com/CiscoDevNet/ncc/blob/master/ncc-get-all-schema but this should have worked (probably).

**Comment by einarnn** [comment]
Yeah, this is an issue. @pakkitzoomzoom, would you like to propose a PR to address this? I think it would be fairly simple to fix.

**Comment by pakkitzoomzoom** [comment]
thanks for confirming the issue. yeah, i'll get to work on one. need it for stuff anyway.
